<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>571</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string>Image</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="imagelabel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>261</width>
      <height>231</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="image">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>571</width>
      <height>351</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>140</y>
      <width>581</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string>Base d'images</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="charger">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>511</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Charger et afficher la base de donn√©es</string>
    </property>
   </widget>
   <widget class="QPushButton" name="indexer">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>70</y>
      <width>521</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculer les descripteurs</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_SIFT">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>85</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>SIFT</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>180</y>
      <width>581</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>1161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_ORB">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>85</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ORB</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_HistC">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>30</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Hist Couleur</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_HSV">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>30</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Hist HSV</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_GLCM">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>30</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>GLCM</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_LBP">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>30</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>LBP</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_HOG">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>30</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>HOG</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quit_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Quitter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="menu_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Menu</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
